bitrise.yml

```diff
@@ -402,6 +402,7 @@ workflows:
 
             # Check for specific applications and set flags accordingly
             for app in $APPS_AFFECTED; do
+                echo "$app"
```

Comment:
The change in this line seems to trigger the `build_applications` step in the pipeline. I will remove this.